libs/orders/src/lib/utils/get-default-order.ts

```diff
@@ -5,28 +5,25 @@ import {
 } from '@vegaprotocol/wallet';
 import { toDecimal } from '@vegaprotocol/react-helpers';
 import type { Market } from '../market';
+import type { OrderStatus } from '@vegaprotocol/types';
 
-export type Order =
-  | {
-      size: string;
-      type: VegaWalletOrderType.Market;
-      timeInForce: VegaWalletOrderTimeInForce;
-      side: VegaWalletOrderSide;
-      price?: never;
-      expiration?: never;
-    }
-  | {
-      size: string;
-      type: VegaWalletOrderType.Limit;
-      timeInForce: VegaWalletOrderTimeInForce;
-      side: VegaWalletOrderSide;
-      price?: string;
-      expiration?: Date;
-    };
+export type Order = {
```

Comment:
question (blocking): Why combine into one type here? I think it reduces type safety between limit and market orders. They were originally separated so that we'd get field discrimination (eg price) between limit and market orders.

Reply:
oh, I see. updated.